### PR TITLE
load balancer file

### LINKED ISSUE #27: Load Balancer
Objectives:
- Create a load balance that only listen to port 443
- Requests to the load balance must be redirected to the Containers running on the ECS cluster
- Load balancer must be publicly accessible
- Security group should only allow TCP connection on port 443

Acceptance Criteria:
By accessing https://loadbalanceurl I can get a response from the container

### PR DESCRIPTION
**Pull request by drii-cavalcanti**
Application load balance configuration file

### CODE REVIEW

File: terraform/lb.tf [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# target group
+resource "aws_alb_target_group" "target-group-alb" {
+    name = "alb-group"
+    port = 443
+    protocol = "HTTPS"
+    vpc_id = "${aws_vpc.da-wordpress-vpc}"
+}
+
+
+##### SG #####
```
Comment (starkmatt) [inline terraform/lb.tf]: Would you be able to create different files for these resources ?
For sg.tf

If you pull from master,
you'll be able to add this code to the existing sg.tf

File: terraform/lb.tf [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+resource "aws_security_group_rule" "sg-alb" {
+  type              = "ingress"
+  from_port         = "443"
+  to_port           = "443"
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = "${aws_security_group.sg-alb.id}"
+}
+
+##### route 53 #####
```
Comment (starkmatt) [inline terraform/lb.tf]: Same for this also!

**Review by starkmatt — COMMENTED** [review]
Hey Adriana,
Great job.

Are you able to split the different resources up into it's relevant file.
For security groups, add them into sg.tf

As for route53,
It'll probably be better to create a route53.tf and add the resource into that!

File: terraform/lb.tf (hunk 2 above)
Comment (drii-cavalcanti) [reply terraform/lb.tf]: done! ;)

File: terraform/lb.tf (hunk 1 above)
Comment (drii-cavalcanti) [reply terraform/lb.tf]: done! ;)

**Review by deniseddi — APPROVED** [review]
Nice!

**Pull request merged**